bring/src/main/java/com/bobocode/petros/bring/exception/BringException.java

```diff
@@ -0,0 +1,12 @@
+package com.bobocode.petros.bring.exception;
+
+/**
+ * General framework-level exception. In case new exception should be registered within the codebase it should
+ * extend this exception.
+ */
+public class BringException extends RuntimeException {
+
+    public BringException(final String message) {
```

Comment:
It is always good to have a constructor with a Throwable parameter, so you could pass a cause exception in case of rethrowing BringException instead of checked exception for example

Reply:
yup, you're right and I think we may need it

Reply:
added additional constructors